apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestDataPathsActivity.java

```diff
@@ -141,7 +157,7 @@ public class TestDataPathsActivity  extends BaseAutoGlitchActivity {
     private static final int[] SHORT_OUTPUT_CHANNEL_MASKS = {
             StreamConfiguration.CHANNEL_MONO,
             StreamConfiguration.CHANNEL_STEREO,
-            StreamConfiguration.CHANNEL_2POINT1,
+            StreamConfiguration.CHANNEL_2POINT1, // TODO should this be 5.1?!
```

Comment:
We can add 5.1 as well. I wanted to add 2.1 as its the smallest case with more than two channels.

Reply:
Added 5.1 and comments.